app/hooks/useClaimTo.tsx

```
@@ -0,0 +1,94 @@
+
+      const timeoutPromise = new Promise((_, reject) => {
+        setTimeout(
+          () => reject(new Error('Transaction timeout')),
+          TRANSACTION_TIMEOUT,
+        );
+      });
+
+      const result = await Promise.race([
+        sendTransaction({
+          transaction,
+          account: activeAccount!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)